docs/gsoc.rst

```diff
@@ -0,0 +1,100 @@
+Google Summer of Code
+=====================
+
+Read the Docs is hoping to participate in the Google Summer of Code in 2018.
+This page will contain all the information for students and anyone else interested in helping.
+
+Required Skills
+---------------
+
+Incoming students will need the following skills:
+
+* Intermediate python programming
+* Familiarity with Markdown, reStructuredText, or some other plain text markup language
+* Familiarity with git, or some other source control
+* Ability to set up your own development environment for Read the Docs
+* An interest in documentation and improving open source documentation tools would be great too!
```

Comment:
I think there should be something like
`Having some JS/CSS knowledge would be awesome things to have`